ansible/files/gotrue.service.j2

```diff
@@ -13,6 +13,7 @@ MemoryAccounting=true
 MemoryMax=50%
 
 EnvironmentFile=/etc/gotrue.env
+EnvironmentFile=/etc/gotrue.override.env
```

Comment:
For consistency with other files with overrides, use plural `/etc/gotrue.overrides.env`